src/plugins/intel_cpu/src/nodes/rnn.cpp

```diff
@@ -232,6 +232,10 @@ bool RNN::isSupportedOperation(const std::shared_ptr<const ov::Node>& op, std::s
                 errorMessage = "Node expects constants as W, R, B inputs.";
                 return false;
             }
+            if (ov::is_type<ov::op::v0::LSTMCell>(op) && op->get_input_size() == 7) {
```

Comment:
Let's align it with the rest of the error messages
```suggestion
            if (ov::is_type<ov::op::v0::LSTMCell>(op) && op->get_input_size() != 6) {
                errorMessage = "Node expects 6 inputs. Actual: " + std::to_string(op->get_input_size());
                return false;
            }

```